guides/voice/gather-dtmf-tones-guide/gather-example-step-0/example.5.x.cs

```diff
@@ -0,0 +1,22 @@
+// In Package Manager, run:
+// Install-Package Twilio.Mvc -Pre
+
+using System.Web.Mvc;
+using Twilio.TwiML;
+
+public class VoiceController : Controller
+{
+    [HttpPost]
+    public ActionResult Index()
+    {
+        var response = new VoiceResponse();
+
+        // Use the <Gather> verb to collect user input
+        response.Gather(new Gather(numDigits: 1)
+		                .Say("For sales, press 1. For support, press 2."));
```

Comment:
That's true, but I want to emphasize that _Say_ is verb nested into _Gather_. I do prefer the current alignment.